src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/useOpprettBehandling.ts

```diff
@@ -21,6 +21,10 @@ const useOpprettBehandling = (lukkModal: () => void) => {
     const [selectedBehandlingÅrsak, settSelectedBehandlingÅrsak] = useState<BehandlingÅrsak | ''>(
         ''
     );
+    const [visValideringsFeil, settVisValideringsfeil] = useState({
+        behandlingstype: false,
+        behandlingÅrsak: false,
```

Comment:
Kunne man kanskje satt feilmeldingene her istedet? Da ville man fått en feilmelding per input, samt en feilmelding for innsending(submitRessurs)

Reply:
Ja, tenker du sånn? (siste commit)

Reply:
Ja 👍

Reply:
oki :) er resten greit?